install.sh

```diff
@@ -172,10 +170,13 @@ _dirname() {
 # Result: print full path
 ###################################################
 _full_path() {
-    case "${1?${FUNCNAME[0]}: Missing arguments}" in
-        /*) printf "%s\n" "${1}" ;;
-        *) printf "%s\n" "${PWD}/${1}" ;;
-    esac
+    [[ $# = 0 ]] && printf "%s: Missing arguments\n" "${FUNCNAME[0]}" && return 1
+    declare input="${1}"
+    if [[ -f ${input} ]]; then
+        printf "%s/%s\n" "$(cd $(_dirname "${input}") && pwd)" "${input##*/}"
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Quote this to prevent word splitting.](https://app.codacy.com/manual/labbots/google-drive-upload/pullRequest?prid=5571879)